.github/workflows/assets-test.yaml

```
@@ -138,7 +138,19 @@ jobs:
           tenant-id: ${{ secrets.AZURE_TENANT_ID }}
           subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
           scripts-setup-dir: ${{ env.scripts_setup_dir }}
-      
+
+      - name: Fetch OID token every 4 mins
```

If you want to keep "OID" terminology, please refer to it as "OIDC" instead, or consider this suggestion.
```suggestion
      - name: Refresh login every 4 minutes
```